From 164e0dc8d05f4a18627600abb1a8c2e38d905d83 Mon Sep 17 00:00:00 2001
From: Jon Nordby <jononor@gmail.com>
Date: Wed, 2 Feb 2011 00:22:54 +0200
Subject: [PATCH] Build: Install python code to DESTDIR

---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index f5dd8d7..4b30453 100644
--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ test:
 	cd tests/; python alltest.py
 
 install: all install-data
-	python setup.py install
+	python setup.py install --root=${DESTDIR}
 
 develop: all install-data
 	python setup.py develop
-- 
1.7.4

